### PR TITLE
[Frontend]: Cart table component(see notes)

### PR DESCRIPTION
**Pull request by brentmillen**
### [Cart table component](https://trello.com/c/wumTBHXD/84-display-cart-items-component)
  _Other tickets with changes:_
  - [Cart drop down component in header](https://trello.com/c/u7CDrYvL/52-cart-dropdown-view)
  - [Quantity component](https://trello.com/c/L21PspvK/99-increase-decrease-item-quantity-component)

## Changes

- Create cart table in temporary cart view
- Cart row component for each item
- Utilize quantity +/- component
- Responsize cart using defined breakpoints
Other:
- Set default breakpoints in theme
- Add import for bold font
- Add responsiveness to quantity +/- component

## Screenshot

cartpageweb
image

## How to test?

- checkout to branch
- pnpm run dev
- http://localhost:5173/cart
_notes: 
-also check on cart dropdown for the updated quantity component
-confirm if the small device design is also approved_

### CODE REVIEW

**Comment by markjoshuacarmen** [comment]
image

File: apps/web/src/views/CartView.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import CartTable from '../../../../packages/ui/cart/CartTable';
+export const CartView = () => {
+  return (
+    <main>
+      {/* Section height is for demo purposes only; please position the footer at the bottom instead of the middle */}
+      <section style={{ height: '100vh', background: 'lightgray' }}>
```
Comment (SGVGen) [inline apps/web/src/views/CartView.tsx:8]: Use percentage than vh (viewheight)
```suggestion
      <section style={{ height: '100%', background: 'lightgray' }}>
```

**Comment by TreshAnn** [comment]
everything's running smoothly on my end. Just wanted to drop in a suggestion about the cart table. Keep the header locked in place at the top of the product list. As for the products list, they should be the only things that scroll within their designated area.

**Comment by brentmillen** [comment]
> image

fixed, thanks

**Comment by brentmillen** [comment]
> everything's running smoothly on my end. Just wanted to drop in a suggestion about the cart table. Keep the header locked in place at the top of the product list. As for the products list, they should be the only things that scroll within their designated area.

been trying to apply this but no luck so far. either the row component gets rendered on the first column only or the space between the header and body gets messed up(mantine table). feel free to apply changes in this branch regarding this.

**Comment by SGVGen** [comment]
Try this @brentmillen for sticky Header: https://ui.mantine.dev/component/table-scroll-area

Thank you!

**Review by sandrex-SGV — COMMENTED** [review]
suggestion: what if the cart's title bar does not move along with the cart items when scrolling.

reference: [CSS Position - Sticky](https://www.w3schools.com/cssref/tryit.php?filename=trycss_position_sticky)

**Comment by brentmillen** [comment]
Thank you for your references. Was able to apply it before but it breaks the design. I re-applied it in this latest [commit](https://github.com/dandalandone/POC-EyCommerce/pull/70/commits/dae21363c99c5455a30deec6c22f55673963eb04). Kindly check if this will do for the mean time.
@SGVGen @5andrex

**Review by SGVGen — APPROVED** [review]
LGTM 

image

**Pull request merged**